voting/src/lib.rs

```diff
@@ -47,13 +47,13 @@ impl VotingContract {
         assert!(self.result.is_none(), "Voting has already ended");
         let cur_epoch_height = env::epoch_height();
         if cur_epoch_height != self.last_epoch_height {
-            for account_id in self.votes.keys().into_iter().collect::<Vec<_>>() {
+            let votes = std::mem::take(&mut self.votes);
+            for (account_id, account_voted_stake) in votes.into_iter() {
```

Comment:
`into_iter` is unnecessary

Reply:
Good point. It produced the same binary.